### PR TITLE
翻译development/hardware/usb_setup.md

### PR DESCRIPTION
**Pull request by Zengruiray**
更新了development/hardware/usb_setup.md的中文翻译

### CODE REVIEW

File: development/hardware/usb_setup.md [hunk 1]
```diff
@@ -1,54 +1,66 @@
-# Install Fuchsia from a USB flash drive
+<!--# Install Fuchsia from a USB flash drive-->
+# 从 USB 闪存驱动器安装Fuchsia
```
Comment (ctzsm) [inline development/hardware/usb_setup.md:2]:
```suggestion
# 从 USB 闪存驱动器安装 Fuchsia
```

File: development/hardware/usb_setup.md [hunk 2]
```diff
@@ -1,54 +1,66 @@
...
+<!--1. Create a bootable USB drive:-->
+1. 请创建一个可启动的 USB 驱动器：
 
    ```posix-terminal
    fx mkinstaller -v --new-installer {{ "<var>" }}PATH_TO_USB_DRIVE{{ "</var>"}}
    ```
 
-   Replace `PATH_TO_USB_DRIVE` with the path to the USB drive from the step above.
+   <!--Replace `PATH_TO_USB_DRIVE` with the path to the USB drive from the step above.-->
+   请将 `PATH_TO_USB_DRIVE` 替换为上述步骤中.
```
Comment (ctzsm) [inline development/hardware/usb_setup.md:53]:
```suggestion
   请将 `PATH_TO_USB_DRIVE` 替换为上述步骤中得到的 USB 驱动器路径。
```

File: development/hardware/usb_setup.md [hunk 3]
```diff
@@ -1,54 +1,66 @@
...
 
-   The example command below selects the `/dev/sda` path:
+   <!--The example command below selects the `/dev/sda` path:-->
+   请在下面的示例命令中选择 `/dev/sda` 路径：
```
Comment (ctzsm) [inline development/hardware/usb_setup.md:56]:
```suggestion
   下面的示例命令选取了 `/dev/sda` 路径：
```

File: development/hardware/usb_setup.md [hunk 4]
```diff
@@ -62,22 +74,32 @@ To prepare a USB flash drive to be a bootable disk, do the following:
...
+   <!--The device boots into the Fuchsia Workstation Installer.-->
+   设备启动到 Fuchsia Workstation 安装程序。
 
-1. Press **Enter** on prompts to continue the installation process.
+<!--1. Press **Enter** on prompts to continue the installation process.-->
+1. 请在出现提示时按 **Enter** 以继续安装过程。 
 
-   When the installation is finished, the screen displays `Success! Please restart your computer.`
+   <!--When the installation is finished, the screen displays `Success! Please restart your computer.`-->
+   在安装完成后，屏幕显示`Success! Please restart your computer.`
```
Comment (ctzsm) [inline development/hardware/usb_setup.md:96]:
```suggestion
   在安装完成后，屏幕显示 `Success! Please restart your computer.`
```

File: development/hardware/usb_setup.md [hunk 5]
```diff
@@ -62,22 +74,32 @@ To prepare a USB flash drive to be a bootable disk, do the following:
...
+<!--1. Unplug the USB drive from the target device.-->
+1. 请从目标设备上拔下 USB 驱动器。
 
-1. Reboot the target device.
+<!--1. Reboot the target device.-->
+1. 请重新启动目标设备。
 
-   The target device is now booted into Fuchsia’s Workstation.
+   <!--The target device is now booted into Fuchsia’s Workstation.-->
+   随后目标设备现在启动到 Fuchsia 的工作站（Workstation）。
```
Comment (ctzsm) [inline development/hardware/usb_setup.md:105]:
```suggestion
   目标设备现在会启动到 Fuchsia 的工作站（Workstation）。
```

File: development/hardware/usb_setup.md [hunk 6]
```diff
@@ -1,54 +1,66 @@
...
+   然后请用此命令打印出类似于以下内容的输出：
 
    ``` none {:.devsite-disable-click-to-copy}
    $ fx list-usb-disks
    /dev/sda - My Example USB Disk
    ```
 
-1. Create a bootable USB drive:
+<!--1. Create a bootable USB drive:-->
+1. 请创建一个可启动的 USB 驱动器：
```
Comment (ctzsm) [inline development/hardware/usb_setup.md:46]:
```suggestion
1. 创建一个可启动的 USB 驱动器：
```

File: development/hardware/usb_setup.md (hunk 3 above)
Comment (Zengruiray) [reply development/hardware/usb_setup.md:56]: OK

File: development/hardware/usb_setup.md (hunk 1 above)
Comment (Zengruiray) [reply development/hardware/usb_setup.md:2]: 您好，不好意思，前段时间学业繁重，我马上修改

**Pull request merged**